src/btree/row_srch.c

```diff
@@ -173,10 +176,14 @@ __wt_search_insert(
 
         if (cmp > 0) { /* Keep going at this level */
             insp = &ins->next[i];
+            WT_ASSERT_ALWAYS(session, match >= skiplow,
```

Comment:
I don't think this assert makes sense unless we compare the keys from the start. Otherwise, this assert will always be true.

Reply:
Yep, I'll drop it.

This was an artifact from back when we first found the OOO issue. It can technically happen when we're searching through a skiplist and there is an incorrect `next_stack`. For example if `skiplow == 0` while we're searching for `AAABC`. We see key `AAADD` so `skiphigh = 3`, then move down the a level and (incorrectly) see `AAD`. We'll compare the whole string due to `skiplow == 0` and then change `skiphigh` from 3 down to 2 hitting the assert.